content/webapp/utils/iiif/v3/index.ts

```diff
@@ -806,7 +806,7 @@ export function hasNonImages(
   return !!hasNonImage;
 }
 
-export function getFormatString(format: string): string | undefined {
+export function getFormatString(format?: string): string | undefined {
```

Comment:
I think this will never return `undefined` as the default case is set.

And a nitpick/tidy is to amend where we've done `getFormatString(displayItem.format || '')` since now `format` can be `undefined`, it makes it unclear.